src/main/java/com/adobe/commerce/cif/api/CategoryApi.java

```diff
@@ -58,6 +58,21 @@ Category getCategoryById(
         @ApiParam(value = ACCEPT_LANGUAGE_DESC)
         @HeaderParam(ACCEPT_LANGUAGE) String acceptLanguage
     );
+
+    @GET
+    @Path("/slug/{slug}")
```

Comment:
We always use plural for endpoints like `products`, `carts`, so IMO this should be `slugs`

Reply:
No sure about that, slug (similar like id) is to identify exactly one.